addons/html_editor/static/tests/_helpers/editor.js

```diff
@@ -95,7 +95,7 @@ export async function setupEditor(content, options = {}) {
     const styleContent = options.styleContent || "";
     await mountWithCleanup(TestEditor, {
         props: {
-            content,
+            content: markup(content),
```

Comment:
As discussed ok for the blind markup here, but it should have a comment above it to explain why it is acceptable.